cdap-ui/app/features/adapters/etlapps.less

```diff
@@ -125,25 +122,317 @@ body.theme-cdap.state-adapters-create {
 
   .modal {
 
+    .modal-dialog {
+      margin-top: 115px;
+    }
     .modal-lg {
       width: 95%;
     }
+    .modal-header {
+      background-color: @body-bg;
+      color: @cdap-header;
+      height: 60px;
+      padding: 0;
+      .border-radius(4px 4px 0 0);
+      div.pull-left {
+        height: 60px;
+        padding-left: 15px;
+        max-width: 725px;
+        * {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          white-space: nowrap;
+        }
+      }
+      h3.modal-title {
+        font-weight: 500;
+        line-height: 1.3;
+        padding-top: 5px;
+      }
+      p {
+        color: @cdap-lightgray;
+        margin: 0;
+        max-width: 740px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+      .btn-group {
+        display: block;
+        text-align: right;
+        > div {
+          background-color: transparent;
+          border-left: 1px solid @cdap-lightgray;
+          text-align: center;
+          width: 60px;
+          .transition(all 0.2s ease);
+          &:first-child {
+            .btn {
+              border: 0;
+              .border-radius(0);
+            }
+            &:hover, &:focus {
+              .border-radius(0 4px 0 0);
+            }
+          }
+          &:hover, &:focus {
+            background-color: @cdap-header;
+            .btn {
+              color: white;
+            }
+          }
+        }
+      }
+      .btn {
+        background-color: transparent;
+        color: @cdap-header;
+        display: block;
+        font-size: 18px;
+        line-height: 60px;
+        padding: 0;
+        width: 100%;
+      }
+    }
+    .modal-body {
+      background-color: white;
```

Comment:
When we have more properties/schema fields (output schema) and reduce the height of browser the content beneath gets hidden (in-accessible). May be setting height only on certain screen sizes might be re-thought?